lib/aws/tests/ses.test.js

```diff
@@ -77,4 +77,9 @@ describe('', () => {
     await ses.send('from', 'to', 'subject', { attachments: ['/path/file'] })
     expect(spy).toHaveBeenLastCalledWith(expect.objectContaining({ attachments: [{ filename: 'file', path: '/path/file' }] }))
   })
+  test('send attachment should set attachment with content', async () => {
```

Comment:
@blazblue95 the idea behind path is that the library will be the one to pickup the file which is stored somewhere reachable from the server i.e. local storage

your use case is also valid where the application already has the content and no need for the library to fetch it from anywhere. do you think we should just support both?

Reply:
Yeah i think this will make it more flexible so that maybe people who want to construct their own content in the app can just use it